roles/edpm_frr/tasks/install.yml

```diff
@@ -25,7 +26,20 @@
     - {'path': "{{ edpm_frr_config_basedir }}", 'setype': container_file_t, 'mode': '0750'}
     - {'path': /run/frr, 'setype': container_file_t, 'mode': '0755'}
 
+- name: Create persistent directories (just for testing purpose)
```

Comment:
if this is needed just for testing only, shouldn't it be on the prepare.yaml instead?

Reply:
moved, make sense!